js/dragquestion.js

```diff
@@ -868,6 +900,9 @@ H5P.DragQuestion = (function ($) {
     self.y = element.y;
     self.width = element.width;
     self.height = element.height;
+    if (self.height < 10) {
```

Comment:
Does this need to be changed to a different number now that we have reverted to em units?